### PR TITLE
[docs] update a few docs

### PR DESCRIPTION
**Pull request by celenechang**
### What does this PR do?

Update How to contribute, Getting Started and Installation docs

### Motivation

Better clarity

### Additional Notes

Anything else we should know when reviewing?

### Minimum Agent Versions

Are there minimum versions of the Datadog Agent and/or Cluster Agent required?

* Agent: vX.Y.Z
* Cluster Agent: vX.Y.Z

### Describe your test plan

Write there any instructions and details you may have to test your PR.

### Checklist

- [X] PR has at least one valid label: `bug`, `enhancement`, `refactoring`, `documentation`, `tooling`, and/or `dependencies`
- [X] PR has a milestone or the `qa/skip-qa` label

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Visit the [Integrations Autodiscovery][9] page for details about how to configur
 
 #### Containers registry
 
-To change the default registry ([gcr.io/datadoghq][6]) to another registry, use the option `spec.registry`.
+To change the default registry ([gcr.io/datadoghq][6]) to another registry, use the option `spec.global.registry`.
```
Comment (khewonc) [inline docs/getting_started.md:64]: github ui won't let me make a suggestion for line 93, but the the gcr link is broken: `ttps://gcr.io/datadoghq`

File: docs/installation.md [hunk 2]
```diff
@@ -2,28 +2,74 @@
...
+```
+
+The Helm release can be updated with
 
-The [Operator Lifecycle Manager][4] framework allows overriding default Operator configuration. See the [Subscription Config][6] document for a list of the supported installation configuration parameters.
+```shell
+helm upgrade my-datadog-operator datadog/datadog-operator -f values.yaml
+```
+
+### Add Credentials for DatadogMonitor
```
Comment (jhgilbert) [inline docs/installation.md:36]:
```suggestion
### Add credentials for DatadogMonitor
```

File: docs/installation.md [hunk 3]
```diff
@@ -2,28 +2,74 @@
...
+
+If using the DatadogMonitor controller, the Datadog Operator requires API and application keys to be configured. 
+Create a Kubernetes Secret that contains your API and application keys. In the example below, the secret keys are `api-key` and `app-key`.
+
+```
+export DD_API_KEY=<replace-by-your-api-key>
```
Comment (jhgilbert) [inline docs/installation.md:42]:
```suggestion
export DD_API_KEY=<YOUR_API_KEY>
```

File: docs/installation.md [hunk 4]
```diff
@@ -2,28 +2,74 @@
...
+export DD_APP_KEY=<replace-by-your-app-key>
```
Comment (jhgilbert) [inline docs/installation.md:43]:
```suggestion
export DD_APP_KEY=<YOUR_APP_KEY>
```

File: docs/installation.md [hunk 5]
```diff
@@ -2,28 +2,74 @@
...
 
-The [Operator Lifecycle Manager][4] framework allows overriding default Operator configuration. See the [Subscription Config][6] document for a list of the supported installation configuration parameters.
+```shell
+helm upgrade my-datadog-operator datadog/datadog-operator -f values.yaml
+```
+
+### Add Credentials for DatadogMonitor
+
+If using the DatadogMonitor controller, the Datadog Operator requires API and application keys to be configured. 
+Create a Kubernetes Secret that contains your API and application keys. In the example below, the secret keys are `api-key` and `app-key`.
```
Comment (jhgilbert) [inline docs/installation.md:39]:
```suggestion
Create a Kubernetes Secret that contains your API and application keys.
```

File: docs/installation.md [hunk 6]
```diff
@@ -48,20 +94,19 @@ spec:
         cpu: "500m"
 ```
 
-### 2. Add Datadog credentials to the Operator
-
-The Datadog Operator requires access to your API and application keys to add a [Datadog monitor](https://docs.datadoghq.com/monitors/). 
+### Add Credentials for DatadogMonitor
```
Comment (jhgilbert) [inline docs/installation.md:97]:
```suggestion
### Add credentials for DatadogMonitor
```

File: docs/installation.md [hunk 7]
```diff
@@ -48,20 +94,19 @@ spec:
...
 
-1. Create a Kubernetes Secret that contains your API and application keys. In the example below, the secret keys are `api-key` and `app-key`.
+If using the DatadogMonitor controller, the Datadog Operator requires API and application keys to be configured. 
+Create a Kubernetes Secret that contains your API and application keys. In the example below, the secret keys are `api-key` and `app-key`.
```
Comment (jhgilbert) [inline docs/installation.md:100]:
```suggestion
Create a Kubernetes Secret that contains your API and application keys.
```

File: docs/installation.md [hunk 8]
```diff
@@ -48,20 +94,19 @@ spec:
...
 
 ```
 export DD_API_KEY=<replace-by-your-api-key>
```
Comment (jhgilbert) [inline docs/installation.md:103]:
```suggestion
export DD_API_KEY=<YOUR_API_KEY>
```

File: docs/installation.md [hunk 9]
```diff
@@ -48,20 +94,19 @@ spec:
...
 export DD_APP_KEY=<replace-by-your-app-key>
```
Comment (jhgilbert) [inline docs/installation.md:104]:
```suggestion
export DD_APP_KEY=<YOUR_APP_KEY>
```

File: docs/installation.md [hunk 10]
```diff
@@ -80,27 +125,28 @@ spec:
...
           secretKeyRef: 
             key: app-key
-            name: datadog-secret
+            name: datadog-operator-secret
 ```
 
-**Note**: You can add configuration overrides in the `spec.config` section. For example, override `env` and `resources`.
 
-## 3. Deploy the Datadog Agents with the Operator
+## Deploy the DatadogAgent custom resource managed by the Operator
```
Comment (jhgilbert) [inline docs/installation.md:137]:
```suggestion
## Deploy the `DatadogAgent` custom resource managed by the Operator
```

File: docs/installation.md [hunk 11]
```diff
@@ -80,27 +125,28 @@ spec:
...
 
-After deploying the Datadog Operator, create the `DatadogAgent` resource that triggers the Datadog Agent's deployment in your Kubernetes cluster. The creation of this resource deploys the Agent as a DaemonSet on every Node of your cluster.
+After deploying the Datadog Operator, create the `DatadogAgent` resource that triggers the deployment of the Datadog Agent, Cluster Agent and Cluster Checks Runners (if used) in your Kubernetes cluster. The Datadog Agent will be deployed as a DaemonSet, running a pod on every Node of your cluster.
 
-1. Create a Kubernetes secret with your API and APP keys
+1. Create a Kubernetes secret with your API and App keys
```
Comment (jhgilbert) [inline docs/installation.md:141]:
```suggestion
1. Create a Kubernetes secret with your API and application keys
```

File: docs/installation.md [hunk 12]
```diff
@@ -80,27 +125,28 @@ spec:
...
+            name: datadog-operator-secret
 ```
 
-**Note**: You can add configuration overrides in the `spec.config` section. For example, override `env` and `resources`.
 
-## 3. Deploy the Datadog Agents with the Operator
+## Deploy the DatadogAgent custom resource managed by the Operator
 
-After deploying the Datadog Operator, create the `DatadogAgent` resource that triggers the Datadog Agent's deployment in your Kubernetes cluster. The creation of this resource deploys the Agent as a DaemonSet on every Node of your cluster.
+After deploying the Datadog Operator, create the `DatadogAgent` resource that triggers the deployment of the Datadog Agent, Cluster Agent and Cluster Checks Runners (if used) in your Kubernetes cluster. The Datadog Agent will be deployed as a DaemonSet, running a pod on every Node of your cluster.
```
Comment (jhgilbert) [inline docs/installation.md:139]:
```suggestion
After deploying the Datadog Operator, create the `DatadogAgent` resource that triggers the deployment of the Datadog Agent, Cluster Agent and Cluster Checks Runners (if used) in your Kubernetes cluster. The Datadog Agent will be deployed as a DaemonSet, running a pod on every node of your cluster.
```

File: docs/installation.md [hunk 13]
```diff
@@ -80,27 +125,28 @@ spec:
...
 
-1. Create a Kubernetes secret with your API and APP keys
+1. Create a Kubernetes secret with your API and App keys
 
    ```shell
+   export DD_API_KEY=<replace-by-your-api-key>
```
Comment (jhgilbert) [inline docs/installation.md:144]:
```suggestion
   export DD_API_KEY=<YOUR_API_KEY>
```

File: docs/installation.md [hunk 14]
```diff
@@ -80,27 +125,28 @@ spec:
...
+   export DD_APP_KEY=<replace-by-your-app-key>
```
Comment (jhgilbert) [inline docs/installation.md:145]:
```suggestion
   export DD_APP_KEY=<YOUR_APP_KEY>
```

**Review by jhgilbert — APPROVED** [review]
Approved with some suggestions for style and consistency. Thank you!

File: docs/installation.md [hunk 15]
```diff
@@ -2,28 +2,74 @@
...
+The Helm release can be updated with
+
+```shell
+helm upgrade my-datadog-operator datadog/datadog-operator -f values.yaml
+```
+
+
+### Install the Datadog Operator with Operator Lifecycle Manager
+
+Instructions for deploying the Datadog Operator with [Operator Lifecycle Manager][4] is available at [operatorhub.io][5].
```
Comment (khewonc) [inline docs/installation.md:68]:
```suggestion
Instructions for deploying the Datadog Operator with [Operator Lifecycle Manager][4] are available at [operatorhub.io][5].
```

**Pull request merged**